plugins/lang-painless/src/main/java/org/elasticsearch/painless/PainlessLexer.java

```diff
@@ -1,19 +1,16 @@
 // ANTLR GENERATED CODE: DO NOT EDIT
 package org.elasticsearch.painless;
 
-import org.antlr.v4.runtime.CharStream;
+    import java.util.Set;
```

Comment:
You can ```<suppress checks="." files="plugins/lang-painless/src/main/java/org/elasticsearch/painless/PainlessLexer.java"/>``` or whatever pattern we need on the end to get any other auto-generated files (something like that, going off-the-cuff here).